src/main/resources/db/migration/V1.0.0__create_songs_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS songs(
+ id INTEGER NOT NULL GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+ artistName VARCHAR(255) NOT NULL,
+ trackName VARCHAR(255) NOT NULL,
+ trackTime TIME NOT NULL,
```

Comment:
To typ danych reprezentujących czas, tj. godziny i minuty w zakresie 00:00-23:59 w oderwaniu od daty - o ile oczywiście dałoby się tego użyć w założony przez Ciebie sposób, to jednak wydaje mi się, że nie do końca tu pasuje.

Nazwałbym to chyba `trackLength` i do reprezentacji użył albo typu `interval` (mapowane na `Duration` - ale tu prawdopodobnie [trzeba trochę Hibernate pomóc](https://vladmihalcea.com/map-postgresql-interval-java-duration-hibernate/)) lub po prostu `int`/`bigint` mapowanych odpowiednio na `int`/`long` (jeśli to czas w sekundach to `int`  zupełności wystarczy, jeśli w milisekundach to konwencja raczej zaleca stosowanie `long`).

Zgodnie z zasadą KISS osobiście pewnie wybrałbym `int` reprezentujący po prostu długość utworu w sekundach, ale ostateczna decyzja należy do Ciebie 😉